tests/platform_tests/conftest.py

```diff
@@ -14,6 +16,26 @@ def skip_on_simx(duthosts, rand_one_dut_hostname):
     if "simx" in platform:
         pytest.skip('skipped on this platform: {}'.format(platform))
 
+@pytest.fixture(scope="module")
+def xcvr_skip_list(duthosts, rand_one_dut_hostname):
+    duthost = duthosts[rand_one_dut_hostname]
+    platform = duthost.facts['platform']
+    hwsku = duthost.facts['hwsku']
+    f_path = os.path.join('/usr/share/sonic/device', platform, hwsku, 'hwsku.json')
+    intf_skip_list = []
+    try:
+        out = duthost.command("cat {}".format(f_path))
+        int_skip_info = json.loads(out["stdout"])
+        for int_n in int_skip_info['interfaces']:
+            if int_skip_info['interfaces'][int_n]['port_type'] == "RJ45":
+                intf_skip_list.append(int_n)
```

Comment:
With above naming change, also suggest to change this to:

```python
for interface in hwsku_info['interfaces']:
    if interface['port_type'] == "RJ45":
        intf_skip_list.append(int_n)
```

Reply:
@jleveque  
the suggestion will not work since interface is key interface['port_type'] will not be valid but can do this:
```
 for interface, interface_value in hwsku_info['interfaces'].items():
       if interface_value['port_type'] == "RJ45":
           intf_skip_list.append(interface)

```

Reply:
Ah, right. No worries, in that case just go with:

```python
for interface in hwsku_info['interfaces']:
    if hwsku_info['interfaces'][interface]['port_type'] == "RJ45":
        intf_skip_list.append(interface)
```

Reply:
@jleveque 
made these changes in my latest commit